### PR TITLE
(feature) Strategy Settings Menu (HIDDEN Leverage and Orders Tabs)

### PR DESCRIPTION
**Pull request by dmytroshch**
### Asana task:
https://app.asana.com/0/1201849173362898/1202881975385823
https://app.asana.com/0/1201849173362898/1202881975385832
https://app.asana.com/0/1201848935859401/1202907567859180

### Description:
- refactored `StrategyEditor` and `StrategySettingsModal` (ex `ExecutionOptionsModal`)
- fixed orthographic mistakes
- refactor in i18n storage"

Снимок экрана от 2022-09-16 20-10-51
Снимок экрана от 2022-09-16 20-09-58
8b27-5c7809520e65.png)
Снимок экрана от 2022-09-16 20-09-50
Снимок экрана от 2022-09-16 20-03-09

### Related Pull Requests:
...

### Backend Dependencies:
...

### Other Dependencies:
...

### CODE REVIEW

File: src/components/StrategyEditor/StrategyEditor.js [hunk 1]
```diff
@@ -630,6 +620,11 @@ const StrategyEditor = (props) => {
     [backtestResults],
   )
 
+  const sbtitleSettings = useCallback(
+    ({ sidebarOpened }) => <SettingsTitle sidebarOpened={sidebarOpened} />,
+    [],
+  )
+
```
Comment (bhoomij) [inline src/components/StrategyEditor/StrategyEditor.js:627]: I think it's usecase for useMemo and not useCallback

File: src/modals/AppSettingsModal/AppSettingsModal.constants.js [hunk 2]
```diff
@@ -1,11 +1,25 @@
...
+
+export const SETTINGS_TABS_COMPONENTS = {
+  [SETTINGS_TABS.Beta]: <AppSettingsModalBeta />,
+  [SETTINGS_TABS.General]: <AppSettingsModalGeneral />,
+  [SETTINGS_TABS.Keys]: <AppSettingsModalApiKeys />,
+  [SETTINGS_TABS.AppSettings]: <AppSettingsModalAppSettings />,
+  [SETTINGS_TABS.About]: <AppSettingsModalAbout />,
 }
 
 export const WEB_SETTINGS_TABS = [SETTINGS_TABS.AppSettings, SETTINGS_TABS.About]
```
Comment (bhoomij) [inline src/modals/AppSettingsModal/AppSettingsModal.constants.js:25]: we don't need any tabs for web as discussed earlier over slack DM

**Review by bhoomij — CHANGES_REQUESTED** [review]
@dmytroshch looks good, requested minor changes. for hosted please check current staging we are not showing settings tab so let's keep it same.
Also please create PR for only single task (if possible), at least for bug it should be separate as it becomes difficult to review changes in big PR with lots of changes.

**Review by bhoomij — APPROVED** [review]

**Comment by bhoomij** [comment]
thanks, will merge this after 3.31 release as its not needed for tomorrow release

**Comment by dmytroshch** [comment]
@bhoomij it is related to this task - https://app.asana.com/0/1201849173362898/1202881975385823

**Pull request merged**